fets/README.md

```diff
@@ -1,5 +1,15 @@
 # MedPerf - MLCube - Federated Tumor Segmentation Challenge
 
+This repository contains three [MLCubes](https://github.com/mlcommons/mlcube) used throughout the organization of the FeTS challenge:
+
+TODO Maybe we should even remove the two other containers from the FeTS example?
+
+1. Preprocessing: This MLCube contains the preprocessing pipeline employed in FeTS (which is identical to BraTS).
+2. Model: This MLCube can be used by **task-2 participants** to build their inference application. If you are looking for how to prepare your FeTS task-2 submission, please continue reading [here](model).
+3. Metrics: This MLCube computes the segmentation metrics established in BraTS for (prediction, reference segmentation)-pairs.
+
+TODO Do we need the stuff below?
```

Comment:
Yes, if we keep the preprocessing and metrics containers, it makes sense to also keep MedPerf stuff. Since I added a note that the model cube is the only relevant one for challenge participants, it should be fine to keep it without further changes.